cluster/charts/rook-ceph/templates/resources.yaml

```diff
@@ -226,6 +230,8 @@ spec:
                   replicated:
                     properties:
                       size:
+                        maximum: 10
```

Comment:
@rajatsingh25aug Could you make all of these consistent by adding them in this order? It's just strange to see them switched sometimes.
```
minimum: 1
maximum: 10
```

Reply:
@travisn Please check!